core/src/main/java/nl/nn/adapterframework/extensions/javascript/Rhino.java

```diff
@@ -15,6 +15,7 @@
 */
 package nl.nn.adapterframework.extensions.javascript;
 
```

Comment:
When you work on a class you have to update the copyright notice. And since we change name `Integration Partners` can be changed with `WeAreFrank!`